tests/unit/queryBuilder/QueryBuilder.js

```diff
@@ -1881,8 +1881,12 @@ describe('QueryBuilder', () => {
   });
 });
 
+const factoryBuilder = new QueryBuilder(Model);
+
 function createFindOperation(builder, whereObj) {
-  let method = new QueryBuilderOperation('find');
+  const method = factoryBuilder._findOperationFactory(builder);
+
+  method.onBefore2 = method.onAfter2 = null;
```

Comment:
Getting these tests to work again needed some trickery, since they were not actually using the right `QueryBuilderOperation` subclass. I'm accessing the private `_findOperationFactory()` & co. methods, and then erasing `method.onBefore2 = method.onAfter2 = null;`. Not ideal, but I couldn't figure out another way without having to export private classes...